pkg/debugapi/chunk.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package debugapi
+
+import (
+	"net/http"
+
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/gorilla/mux"
+)
+
+func (s *server) hasChunkHandler(w http.ResponseWriter, r *http.Request) {
+	addr, err := swarm.ParseHexAddress(mux.Vars(r)["address"])
+	if err != nil {
+		s.Logger.Debugf("debug api: parse chunk address: %v", err)
+		jsonhttp.BadRequest(w, "bad address")
+		return
+	}
+
+	has, err := s.Storer.Has(r.Context(), addr)
+	if err != nil {
+		s.Logger.Debugf("debug api: localstore has: %v", err)
+		jsonhttp.BadRequest(w, err)
+		return
+	}
+
+	if has {
+		jsonhttp.OK(w, nil)
+		return
+	}
+
+	jsonhttp.NotFound(w, nil)
```

Comment:
I would make the if condition `!has` in order to have happy path not indented, and all other responses indented.